spec/spec.md

```diff
@@ -504,6 +504,16 @@ While ION supports any public key JWK representation in a DID Document, implemen
 Note: This profile leverages JWT for signature generation and verification only. There is a rich offering of Linked Data Cryptographic Suites which are not covered by this iteration of the profile.
 For reference and more information on LD signature suites see the [Linked Data Cryptographic Suite Registry](https://w3c-ccg.github.io/ld-cryptosuite-registry/).
 
+## Credential Schema
```

Comment:
would it make more sense to call this section Example of Credential Schema? 

and then below:
```
Below is an example of a `credentialSubject` schema for a credential of type `WorkplaceCredential`.
```

Reply:
Actually looking at your other PR, it may make sense to move this into Use Cases > Workplace Credential

Reply:
make clear it's non-normative.
keep separate, but make clear they are related